pkgs/applications/science/math/singular/default.nix

```diff
@@ -13,12 +13,9 @@
 , doxygen
 , graphviz
 , latex2html
-# upstream generates docs with texinfo 4. later versions of texinfo
-# use letters instead of numbers for post-appendix chapters, and we
-# want it to match the upstream format because sage depends on it.
-, texinfo4
+, texinfo
 , texliveSmall
-, enableDocs ? true
+, enableDocs ? !stdenv.isDarwin
```

Comment:
Yes, I saw that this had been explicitly changed a few commits ago, but the issue is building the Singular docs fails *only when building Sage*. I agree that in the context of only this PR we could switch this back, but I believe this adds additional context ofborg can't provide (and, ideally, it will be fixed in https://github.com/NixOS/nixpkgs/pull/264126).